src/blocks/pomodoro.rs

```diff
@@ -126,15 +126,7 @@ impl Block for Pomodoro {
             State::Started => {
                 if self.elapsed >= self.length {
                     let message = self.message.to_owned();
-                    std::thread::spawn(move || -> Result<()> {
-                        match Command::new("i3-nagbar").args(&["-m", &message]).output() {
-                            Ok(_raw_output) => Ok(()),
-                            Err(_) => {
-                                // We don't want the bar to crash if i3-nagbar fails
-                                Ok(())
-                            }
-                        }
-                    });
+                    Command::new("i3-nagbar").stdout(Stdio::null()).args(&["-m", &message]).spawn().expect("Failed to start i3-nagbar");
```

Comment:
syntax taken from
https://doc.rust-lang.org/std/process/struct.Command.html#method.stdout

also note that this and the block below align differently because of the `rustfmt` rules of the project